src/core/time.jl

```diff
@@ -114,6 +114,10 @@ function Base.:-(ts::TimestepIndex, val::Int)
 	return TimestepIndex(ts.index - val)
 end
 
+function Base.:-(ts::TimestepIndex, val::TimestepIndex) 
+	return TimestepIndex(ts.index - val.index)
```

Comment:
Ah good point, we don't need this anymore.  I had it in there to handle needs of `StepRange` but I gave up on that effort because it stopped making sense and made my own `:` functions so we don't need it anymore.